apps/utils/scripts/run-bfd-server

```diff
@@ -111,13 +115,26 @@ Option defaults:
 EOF
 }
 
-while getopts "d:i:kl:xzZh" option ; do
+# Prompt for test database values
+prompt_for_test_db() {
+   read -p "Host: " db_host
+   read -p "Port: " db_port
+   read -p "Username: " "username"
+   read -sp "Password: " "password"
```

Comment:
Did you test this? RDS tokens appear to be too long for the password input. I think a better approach would be to take the DB parameters in using Bash command line arguments.